webapp/src/hooks/sortable.tsx

```diff
@@ -17,23 +30,23 @@ function useSortableBase(itemType: string, item: any, enabled: boolean, handler:
         collect: (monitor) => ({
             isOver: monitor.isOver(),
         }),
-        drop: (dragItem: any) => {
-            handler(dragItem, item)
+        drop: (dragItem: HandlerBaseItem) => {
+            handler(dragItem, item as HandlerBaseItem)
         },
         canDrop: () => enabled,
     }), [item, handler, enabled])
 
     return [isDragging, isOver, drag, drop, preview]
 }
 
-export function useSortable(itemType: string, item: any, enabled: boolean, handler: (src: any, st: any) => void): [boolean, boolean, React.RefObject<HTMLDivElement>] {
+export function useSortable(itemType: string, item: SortableItem, enabled: boolean, handler: (src: HandlerItem, st: HandlerItem) => void): [boolean, boolean, React.RefObject<HTMLDivElement>] {
```

Comment:
We can Simplify this by using Generics. Something like the following

```suggestion
export function useSortable<T>(itemType: string, item: T, enabled: boolean, handler: (src: T, st: T) => void): [boolean, boolean, React.RefObject<HTMLDivElement>] {
```